src/Storefront/Resources/views/storefront/element/cms-element-image-gallery.html.twig

```diff
@@ -703,30 +731,22 @@
-                                                                                    {% block element_image_gallery_inner_zoom_modal_slider_control_prev_icon %}
-                                                                                        {% sw_icon 'arrow-head-left' %}
-                                                                                    {% endblock %}
-                                                                                </button>
-                                                                            {% endblock %}
-
-                                                                            {% block element_image_gallery_inner_zoom_modal_slider_control_next %}
-                                                                                <button class="base-slider-controls-next gallery-slider-controls-next"
-                                                                                        aria-label="{{ 'general.next'|trans|striptags }}">
-                                                                                    {% block element_image_gallery_inner_zoom_modal_slider_control_next_icon %}
-                                                                                        {% sw_icon 'arrow-head-right' %}
-                                                                                    {% endblock %}
-                                                                                </button>
+                                                                        <button class="base-slider-controls-next gallery-slider-controls-next"
+                                                                            aria-label="{{ 'general.next'|trans|striptags }}"
+                                                                            data-gallery-slider-control-next="true"
+                                                                        >
+                                                                            {% block element_image_gallery_inner_zoom_modal_slider_control_next_icon %}
+                                                                                {% sw_icon 'arrow-head-right' %}
                                                                             {% endblock %}
-                                                                        </div>
+                                                                        </button>
                                                                     {% endif %}
                                                                 {% endblock %}
+
+
+                                                                {# @deprecated tag:v6.7.0 - This block will be removed #}
+                                                                {% block element_image_gallery_inner_zoom_modal_slider_controls %}{% endblock %}
```

Comment:
Hmm changing the content of the blocks and moving elements to other scopes is a major break. We should implement these changes behind the accessibility feature flag. What do you think, @tobiasberge? 🤔